docs/options.md

```diff
@@ -114,7 +114,7 @@ Options object to pass to [`SVG Optimizer`](http://npmjs.com/package/svgo).
 ### Sprite
 The `sprite` object contains the configuration for the generated sprites in the output spritemap.
 
-#### `sprite.prefix` – `'sprite-'`
+#### `sprite.prefix` – `'sprite-' | function(file){}`
```

Comment:
This can go, the string at the end is the default value, it shouldn't specify the options (that's where the object at the top is for)